dev_guide/authorization.adoc

```diff
@@ -0,0 +1,84 @@
+[[dev-guide-authorization]]
+= Authorization
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[checking-if-users-can-create-pods]]
+== Checking if Users Can Create Pods
+
+Using the `scc-review` and `scc-subject-review` options, you can see if an
+individual user, or a user under a specific service account, can create or
+update a pod.
+
+Using the `scc-review` option, you can check if a service account can create or
+update a pod. The command outputs the security context constraints that admit
+the resource.
+
+For example, to check if a user with the
+`system:serviceaccount:projectname:default` service account can a create a pod:
+
+----
+$ oc policy scc-review -z system:serviceaccount:projectname:default -f my_resource.yaml
+----
+
+You can also use the `scc-subject-review` option to check whether a specific
+user can create or update a pod:
+
+----
+$ oc policy scc-subject-review -u <username> -f my_resource.yaml
+----
+
+To check if a user belonging to a specific group can create a pod in a specific
+file:
+
+----
+$ oc policy scc-subject-review -u <username> -g <groupname> -f my_resource.yaml
+----
+
+[[dev-guide-authorization-determining-what-you-can-do]]
+== Determining What You Can Do as an Authenticated User
+
+From within your {product-title} project, you can determine what verbs you
```

Comment:
Maybe worth a link somehow around here back to the Arch guide's Authorization topic, where "verbs" are explained.